WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2804,6 +2813,24 @@ public void onEventMainThread(VideoOptimizer.ProgressEvent event) {
         }
     }
 
+    @SuppressWarnings("unused")
+    public void onEventMainThread(UploadService.UploadMediaRetryEvent event) {
+        if (!isFinishing()) {
```

Comment:
I *think* this can be simplified to:
```
public void onEventMainThread(UploadService.UploadMediaRetryEvent event) {
    if (!isFinishing()
            && mEditorMediaUploadListener != null
            && event.mediaModelList != null) {
        for (MediaModel media : event.mediaModelList) {
            String localMediaId = String.valueOf(media.getId());
            EditorFragmentAbstract.MediaType mediaType = media.isVideo() ? 
                    EditorFragmentAbstract.MediaType.VIDEO : EditorFragmentAbstract.MediaType.IMAGE;
            mEditorMediaUploadListener.onMediaUploadRetry(localMediaId, mediaType);
        }
    }
}
```

Reply:
This is very good! I don't know how I missed the `media.isVideo()` one. Addressed in 0f4a47b